assets/cbtn.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- type: dataset
+  name: CBTN Pediatric Radiology MRI dataset
+  organization: Children's Brain Tumor Network (CBTN)
+  description: CBTN Pediatric Radiology MRI dataset is a multi-institutional pediatric dataset of clinical radiology MRIs by the Children’s Brain Tumor Network
+  created_date: 2023-10-03
+  url: https://arxiv.org/pdf/2310.01413.pdf
+  datasheet: ''
+  modality: MRI exams
+  size: 23K MRI exams
+  sample: []
+  analysis: Evaluated in comparison to other cancer datasets and variety of diagnoses provided.
+  dependencies: [NCI Childhood Cancer Data Initiative, The Cancer Imaging Archive, Cancer Data Service]
```

Comment:
Are these datasets? Or are these institutions providing data? 

The dependencies should refer to other datasets (and, ideally, those datasets should also be made into nodes and get documented)

Reply:
These are all institutions, but unspecified datasets of theirs were used in the creation of the asset. I updated the dependencies to make that more clear, but since I don't know the exact datasets, creating nodes for them might not offer much value.